app-office/moneydance/moneydance-2019.2.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 2018-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop eapi7-ver java-pkg-2 xdg-utils
+
+MY_PN="Moneydance"
+MY_PV="$(ver_cut 1)"
+
+DESCRIPTION="A cross-platform personal finance application"
+HOMEPAGE="https://moneydance.com/"
+SRC_URI="https://infinitekind.com/stabledl/${MY_PV}/${MY_PN}_linux_amd64.tar.gz -> ${P}-amd64.tar.gz"
+
+LICENSE="Apache-1.0 Apache-2.0 BSD CPAL-1.0 CPL-1.0 CSL-2.0 LGPL-2 MIT TIK"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="|| (
+		>=dev-java/openjdk-bin-11.0
+		>=dev-java/openjdk-11.0
+		>=dev-java/oracle-jdk-bin-11.0
+	)
+"
+
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/${MY_PN}"
+
+RESTRICT="bindist fetch mirror"
+
+pkg_nofetch() {
+	elog "Please download ${A} from"
```

Comment:
You could make it more clear that you expect people to rename the downloaded file.

Reply:
Added a better description.